### ISSUE TITLE
'navbar' request failed with error

### ISSUE THREAD

**Issue by wallverb** [Microsoft/TypeScript#10994]
**TypeScript Version:**  Version 2.1.0-dev.20160919

```sh
[Info  - 1:49:06 PM] Using tsserver from location: /Users/.../.nvm/versions/node/v4.2.2/lib/node_modules/typescript/lib/tsserver.js
[Error - 1:49:09 PM] 'navbar' request failed with error.
Error processing request. Cannot read property 'length' of undefined
TypeError: Cannot read property 'length' of undefined
    at addChildrenRecursively (/Users/.../.nvm/versions/node/v4.2.2/lib/node_modules/typescript/lib/tsserver.js:54261:77)
    at rootNavigationBarNode (/Users/.../.nvm/versions/node/v4.2.2/lib/node_modules/typescript/lib/tsserver.js:54124:17)
    at Object.getNavigationBarItems (/Users/.../.nvm/versions/node/v4.2.2/lib/node_modules/typescript/lib/tsserver.js:54096:47)
    at Object.getNavigationBarItems (/Users/.../.nvm/versions/node/v4.2.2/lib/node_modules/typescript/lib/tsserver.js:59788:37)
    at IOSession.Session.getNavigationBarItems (/Users/.../.nvm/versions/node/v4.2.2/lib/node_modules/typescript/lib/tsserver.js:61009:61)
    at Session.handlers.ts.createMap._a.(anonymous function) (/Users/.../.nvm/versions/node/v4.2.2/lib/node_modules/typescript/lib/tsserver.js:60305:50)
    at IOSession.Session.executeCommand (/Users/.../.nvm/versions/node/v4.2.2/lib/node_modules/typescript/lib/tsserver.js:61137:28)
    at IOSession.Session.onMessage (/Users/.../.nvm/versions/node/v4.2.2/lib/node_modules/typescript/lib/tsserver.js:61154:35)
    at Interface.<anonymous> (/Users/.../.nvm/versions/node/v4.2.2/lib/node_modules/typescript/lib/tsserver.js:63343:27)
    at emitOne (events.js:96:13)

```

Works with `Version 2.1.0-dev.20160915` stopped working in `Version 2.1.0-dev.20160916`

**Comment by wallverb** [comment]
Repro seems to be as easy as:
```ts
/**
 * Test
 */
export const Test = {

}
```

**Comment by mhegazy** [comment]
@andy-ms can you take a look.

**Comment by mhegazy** [comment]
@wallverb can you give `typescript@next` a try later tonight.

**Comment by wallverb** [comment]
@mhegazy What time are new `next` versions published to npm usually ? I am still at `Version 2.1.0-dev.20160919`

**Comment by mhegazy** [comment]
looks like the publish task had issues today. it should be published at midnight tonight under `2.0.1-dev20160921`. sorry for the delay.

**Comment by wallverb** [comment]
Thanks, got 20160920 - and all is good - fixed
